Dockerfile

```diff
@@ -1,13 +1,13 @@
 # Keep hub.Dockerfile aligned to this file as far as possible
 ARG JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom"
-ARG APP_INSIGHTS_AGENT_VERSION=2.3.1-SNAPSHOT
+ARG APP_INSIGHTS_AGENT_VERSION=2.5.1-BETA
 
 FROM hmctspublic.azurecr.io/base/java:openjdk-8-distroless-1.0
```

Comment:
Why not latest ? Will reduce duplication in flux

Reply:
Done. I still kept `applicationinsights-agent.jar` under lib directory as it might required for legacy ASE deployment ?